OloEngine/src/Platform/OpenGL/OpenGLRendererAPI.cpp

```
@@ -519,4 +581,172 @@ namespace OloEngine
+
+        if (enabled)
+        {
+            glEnablei(GL_BLEND, attachment);
+        }
+        else
+        {
+            glDisablei(GL_BLEND, attachment);
+        }
+    }
+
+    static GLenum ToGLTextureTarget(RendererAPI::TextureTargetType target)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)